### ISSUE TITLE
MMOCR example from SDK

### ISSUE THREAD

**Issue by pk-cb** [open-mmlab/mmdeploy#79]
Steps completed:
1. SDK Build
2. Example build

While running the ocr demo in `MMDeploy/build/install/example/` I'm getting this error:
```
./ocr: error while loading shared libraries: libmmdeploy_core.so: cannot open shared object file: No such file or directory
```

**Comment by aliml92** [comment]
same here. Would've been great if the problem solved

**Comment by AllentDan** [comment]
Hi, what if we append `-Wl,--disable-new-dtags` to line 23 of  `cmake/MMDeployConfig.cmake.in`?

**Comment by tehkillerbee** [comment]
I don't have this issue, provided I set LD_LIBRARY_PATH according to the documentation.
```
export LD_LIBRARY_PATH=/home/${USER}/MMDeploy/build/install/lib
```

How did you build your SDK?

**Comment by lvhan028** [comment]
fixed in #80
